siteimprove/admin/js/siteimprove.js

```diff
@@ -38,8 +38,41 @@
       this.callback = callback;
       this.common();
     },
-    common: function () {
-      var _si = window._si || [];
+    common: function (url) {
+      const _si = window._si || [];  
+    
+      const getDomCallback = async function () {
+        const newDiv = document.createElement("div");
+        newDiv.setAttribute("id","div_iframe"); 
+        document.body.appendChild(newDiv);
+        let	string_test = "<iframe id='domIframe' src="+ url.concat("&si_preview=1") +" style='height:100vh; width:100%'></iframe>";
```

Comment:
what is this thing doing?

Reply:
- Create a new div
- Set the Id "div_iframe" to it.
- Insert the Iframe inside it.

Reply:
okay okay you got me there :D but why do you do that?

Reply:
and why is it called string_test?

Reply:
I totally forgot to rename it... 
My Bad.

Reply:
Could you please add some comments for what you are doing in there, I still have a hard time understanding what is going on and why.